### PR TITLE
Classwork on 2022-08-03 and homework for 2022-08-06

### PR DESCRIPTION
**Pull request by sryabkov**
# what

Classwork on Aug 3rd and the homework assigned for next class

# why

Practice, practice, practice

### CODE REVIEW

File: code_school/tuple.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# tuple_variable = 1,0,
+
+
+s = "fdfw"
+# print(tuple_variable)
+
+# the difference between these two for statements is that the first
+# for loop prints out the same string a certain amount of times
+# (to be specific, it prints out the string the amount of times equal to the amount of characters in the string)
+# the second loop prints out each individual letter
```
Comment (sryabkov) [inline code_school/tuple.py:13]: You didn't answer the questions:
- what is the datatype of `i`
- what is the datatype of `e`

File: code_school/calculator.py [hunk 2]
```diff
@@ -1,5 +1,10 @@
 from easygui import *
 
+# assignment 1
+# And add an exponent operation
+# Also handle division by zero
```
Comment (sryabkov) [inline code_school/calculator.py:5]: Do this assignment

File: code_school/functions.py [hunk 3]
```diff
@@ -61,7 +61,7 @@ def print_perimeter_area_rectangle_dict(dict: dict):
 print(print_perimeter_area_rectangle_dict(dict))
 
 # assignment 1
-
+# "Perimeter = 40", "Area = 100"
```
Comment (sryabkov) [inline code_school/functions.py:64]: Implement 

```
def(height: int, width: int) -> tuple(str, str):
```

the first string in the tuple is "Perimeter = (value of perimeter)", the second string in the tuple is "Area = (value of area)"

File: code_school/tuple.py (hunk 1 above)
Comment (awesomeprogrammer1) [reply code_school/tuple.py:13]: Completed

File: code_school/calculator.py (hunk 2 above)
Comment (awesomeprogrammer1) [reply code_school/calculator.py:5]: Completed

File: code_school/functions.py (hunk 3 above)
Comment (awesomeprogrammer1) [reply code_school/functions.py:63]: Fixed

**Pull request merged**